src/Config/Parser/AnnotationParser.php

```diff
@@ -219,7 +223,7 @@ private static function classAnnotationsToGQLConfiguration(
 
         if (null !== $gqlType) {
             if (!$gqlName) {
-                $gqlName = $classAnnotation->name ?: $graphClass->getShortName();
+                $gqlName = isset($classAnnotation->name) ? $classAnnotation->name : $graphClass->getShortName();
```

Comment:
$gqlName = $classAnnotation->name ?? $graphClass->getShortName();

Reply:
I tried it first, but phpstan throws an error "Access to an undefined property object::$name" which is strange, because the `??` operator is a shortcut for `isset` and  `!== null`. This is maybe a bug in phpstan, so I just used `isset` directly to resolve the error. I don't want to flood the code with `// @phpstan-ignore-line` comments, because suppressing errors is the last thing we should do. Let's just leave it `isset` until the bug is fixed.